test/test_mps.py

```diff
@@ -12696,6 +12696,65 @@ def test_resize(self):
         sparse_cpu = sparse_cpu.sparse_resize_(torch.Size([4, 5]), sparse_dim=2, dense_dim=0)
         self.assertEqual(sparse, sparse_cpu)
 
+    def test_coalesce(self):
```

Comment:
Can you just enable portions of `test_sparse`? I.e. why it needs to be a separate MPS test?